admin_guide/cluster_management.rst

```diff
@@ -288,10 +288,39 @@ The new shard(s) are created on the same node as the shard(s) from which the ten
 
 Note that :code:`master_move_shard_placement` will also move any shards which are co-located with the specified one, to preserve their co-location.
 
+Connection Management
```

Comment:
Somewhere we should cover that if values need spaces you can use single quotes (and then need to escape quotes and backslashes). This is covered in the libpq connection string documentation.

Reply:
(added)